revista-brasileira-de-ciencia-do-solo.csl

```diff
@@ -339,41 +339,15 @@
             <text variable="note" suffix=". "/>
           </group>
         </else-if>
-        <else-if type="book" match="any">
-          <choose>
-            <if match="any" variable="edition">
-              <group delimiter=". " prefix=", ">
-                <text macro="edition"/>
-                <group delimiter=", ">
-                  <text macro="publisher"/>
-                  <choose>
-                    <if match="any" variable="number-of-pages">
-                      <text variable="number-of-pages" suffix=" p"/>
-                    </if>
-                    <else-if match="any" variable="page">
-                      <text variable="page" suffix=" p"/>
-                    </else-if>
-                  </choose>
-                </group>
-              </group>
-            </if>
-            <else>
-              <group delimiter=". " prefix=". ">
-                <text macro="edition"/>
-                <group delimiter=", ">
-                  <text macro="publisher"/>
-                  <choose>
-                    <if match="any" variable="number-of-pages">
-                      <text variable="number-of-pages" suffix=" p"/>
-                    </if>
-                    <else-if match="any" variable="page">
-                      <text variable="page" suffix=" p"/>
-                    </else-if>
-                  </choose>
-                </group>
-              </group>
-            </else>
-          </choose>
+       <else-if type="book">
+        <group>
```

Comment:
Also, it's better to use

```
        <group delimiter=". " suffix=". ">
         <text macro="author"/>
         <text macro="title"/>
         <text macro="translator"/>
         <text macro="edition"/>
         <text macro="publisher"/>
         <text macro="locators"/>
        </group>
```

instead of 

```
        <group>
         <text macro="author" suffix=". "/>
         <text macro="title" suffix=". "/>
         <text macro="translator" suffix=". "/>
         <text macro="edition" suffix=". "/>
         <text macro="publisher" suffix=". "/>
         <text macro="locators"/>
        </group>
```